### ISSUE TITLE
 Error in the _Imports.razor

### ISSUE THREAD

**Issue by Dsalazar09** [MarvinKlein1508/SignaturePad#27]
Hello Marvin
![p1](https://github.com/MarvinKlein1508/SignaturePad/assets/57144049/ce86b131-cb15-4291-8f3c-ad7bc84d6b02)
![p2](https://github.com/MarvinKlein1508/SignaturePad/assets/57144049/882b3cbe-3764-4dc3-b86b-22182fce1e4e)

 I am using your code, I installed the nuget package but when invoking it in the imports.razor file I get an error, what is the cause?
 
 My solution is Maui Blazor hybrid. I don't know if that influences the error.

Thank you very much for your collaboration.

**Comment by MarvinKlein1508** [comment]
Hi @Dsalazar09 thanks for reaching out! Could you tell me which .NET version you are using?

**Comment by Dsalazar09** [comment]
Hi @MarvinKlein1508 .NET 7
![p3](https://github.com/MarvinKlein1508/SignaturePad/assets/57144049/ea65760d-6608-4341-98cd-898e127dd8d7)

**Comment by MarvinKlein1508** [comment]
@Dsalazar09 the component requires at least .NET 8. You should be able to update this easily. Luckily it's not as painful anymore as back in .NET Framework days.

.NET 7 ist EOL by next month anyways.

**Comment by Dsalazar09** [comment]
Well @MarvinKlein1508 , I'll try to update the project, I hope I don't suffer too much. thank you

**Comment by MarvinKlein1508** [comment]
@Dsalazar09 in case you cannot update yet, you can use the previous version of this package. It should only required .NET 6.

**Comment by Dsalazar09** [comment]
I don't understand, could I use another version of your package?
![p1](https://github.com/MarvinKlein1508/SignaturePad/assets/57144049/72a3c789-9ba2-4597-a6ad-0e8da034694c)

**Comment by MarvinKlein1508** [comment]
@Dsalazar09 yes. Try v7.5.0. This one is compatible with .NET 6

**Comment by Dsalazar09** [comment]
@MarvinKlein1508 This version of the component did not generate an error, but for MAUI Blazor hybrid it did not work. gets the following error:
[chromium] [INFO:CONSOLE(1)] "Exception has been thrown by the target of an invocation.
[chromium]    at System.RuntimeType.CreateInstanceMono(Boolean nonPublic, Boolean wrapExceptions)
[chromium]    at System.RuntimeType.CreateInstanceDefaultCtor(Boolean publicOnly, Boolean wrapExceptions)
[chromium]    at System.Activator.CreateInstance(Type type, Boolean nonPublic, Boolean wrapExceptions)

![p2](https://github.com/MarvinKlein1508/SignaturePad/assets/57144049/84db3ac8-b2e3-4ef5-8433-41db79115dd6)

**Comment by Dsalazar09** [comment]
Thank you very much friend for sharing your code with me, there are several things that I adjusted and now it worked for me.

I deleted this part:
private SignaturePadOptions _options = new SignaturePadOptions
    {
        LineCap = LineCap.Round,
        LineJoin = LineJoin.Round,
        LineWidth = 20
    };
   
I initialized this just as you have it in the image you shared with me and it worked for me.

You are a crack, thank you very much.

Now I will try to save this in Sqlite and then send it to save in an api.
I am very grateful for your help and timely response.
